### ISSUE TITLE
FMREST --> curl communication error: (77) after recent RHEL patch

### ISSUE THREAD

**Issue by bearinaustin** [kdoronzio/fmREST.php#15]
Hello -- Trying to track down the root cause of this issue:

My server applied a patch this morning and afterwards FMREST throws a CURL communication error when trying to call our filemaker server. I ran an undo on the patch and it was working normally again:

>     Installed     rpm-4.11.3-48.el7_9.x86_64                  @rhel-7-server-rpms
>     Installed     subscription-manager-1.24.52-2.el7_9.x86_64 @rhel-7-server-rpms
>     Installed     yum-3.4.3-168.el7.noarch                    @rhel-7-server-rpms
> Packages Altered:
>     Updated nspr-4.34.0-3.1.el7_9.x86_64             @rhel-7-server-rpms
>     Update       4.35.0-1.el7_9.x86_64               @rhel-7-server-rpms
>     Updated nss-3.79.0-5.el7_9.x86_64                @rhel-7-server-rpms
>     Update      3.90.0-2.el7_9.x86_64                @rhel-7-server-rpms
>     Updated nss-softokn-3.79.0-4.el7_9.x86_64        @rhel-7-server-rpms
>     Update              3.90.0-6.el7_9.x86_64        @rhel-7-server-rpms
>     Updated nss-softokn-freebl-3.79.0-4.el7_9.x86_64 @rhel-7-server-rpms
>     Update                     3.90.0-6.el7_9.x86_64 @rhel-7-server-rpms
>     Updated nss-sysinit-3.79.0-5.el7_9.x86_64        @rhel-7-server-rpms
>     Update              3.90.0-2.el7_9.x86_64        @rhel-7-server-rpms
>     Updated nss-tools-3.79.0-5.el7_9.x86_64          @rhel-7-server-rpms
>     Update            3.90.0-2.el7_9.x86_64          @rhel-7-server-rpms
>     Updated nss-util-3.79.0-1.el7_9.x86_64           @rhel-7-server-rpms
>     Update           3.90.0-1.el7_9.x86_64           @rhel-7-server-rpms

Are you aware of any CURL compatibility issues with recent RHEL updates that I can investigate? I know it wasn't the firewall as I toggled it off and on with no improvement. Are there CURL verify settings I can configure in FMREST?

**Comment by kdoronzio** [comment]
I haven't heard of any issues yet. This would be the first.
The only security checks the class file does is to check for SSL. You try
turning off that level of security.  Let me know if that works.

$fm -> secure = false;

Otherwise, I believe we're using pretty standard cURL calls.

Ken


---------------------------------
Ken d'Oronzio
Paradise Partners, Inc.
866-999-4006 x201

**Comment by bearinaustin** [comment]
Sorry -- I ended up doing a long investigation yesterday and it looks like a CURL cert doesn't show up as replaced until after the apache server was cycled. So it was throwing errors. I only saw the error on Filemaker pages, because those were the only ones throwing errors. I think we can close this as a false positive as I have a working solution in our TEST server post restart of apache. Sorry!
